src/app/templates/SeedPhraseInput/SeedWordInput.tsx

```diff
@@ -48,16 +80,59 @@ export const SeedWordInput: FC<SeedWordInputProps> = ({
         ref={inputRef}
         type="password"
         id={id.toString()}
-        value={value}
         onChange={onChange}
+        // delay for ref update works properly
+        onBlur={e => {
+          if (e.relatedTarget === null) {
+            setOnBlur(true);
+          }
+        }}
+        value={value}
+        onFocus={() => setOnBlur(false)}
         onPaste={handlePaste}
         revealRef={revealRef}
         onReveal={onReveal}
         autoComplete="off"
         smallPaddings
         fieldWrapperBottomMargin={false}
         testID={testID}
+        errorCaption={errorCheckRef.current}
+        style={{ backgroundColor: 'white' }}
```

Comment:
can't you use `bg-white` here instead?

Reply:
unfortunately, in tailwind we cannot override classes, and we have another color inside FormField